README.md

```diff
@@ -123,6 +123,49 @@ include:
 Proxy models are always excluded. They are the same as the model they proxy,
 so there is no benefit in including them.
 
+## Anonymizing data
+
+Leukeleu-django-gdpr comes with a `anonymize` management command.
+
+```
+./manage.py anonymize
+```
+
+(You can only run this command when `DEBUG=True`. It's meant to run locally)
+
+This command uses the `gdpr.yaml` file to anonymize all PII fields in the database. 
+By default, it will anonymize **all fields** that contain PII.
+
+To change the configuration, you can create a subclass of `BaseAnonymizer`:
+
+```python
+# some_file.py
+class Anonymizer(BaseAnonymizer):
+    # To skip rows
+    extra_qs_overrides = {
+        "auth.User": User._base_manager.exclude(is_superuser=True),
+        ...
+    }
+
+    # To change what fake data should be used for a field
+    extra_field_overrides = {
+        "auth.User.first_name": fake.first_name,
```

Comment:
Would be nice to mention where `fake` is supposed to come from. Or maybe use an `@property` so `self.fake` can be used? (still mention that it's an instance of `Faker`).

Reply:
oh and also mention that this particular example already is used by default